napari/_vispy/utils/gl.py

```diff
@@ -112,39 +115,39 @@ def fix_data_dtype(data):
 
 
 BLENDING_MODES = {
-    'opaque': dict(
-        depth_test=True,
-        cull_face=False,
-        blend=False,
-        blend_func=('one', 'zero'),
-        blend_equation='func_add',
-    ),
-    'translucent': dict(
-        depth_test=True,
-        cull_face=False,
-        blend=True,
-        blend_func=('src_alpha', 'one_minus_src_alpha', 'zero', 'one'),
-        blend_equation='func_add',
-    ),
-    'translucent_no_depth': dict(
-        depth_test=False,
-        cull_face=False,
-        blend=True,
-        blend_func=('src_alpha', 'one_minus_src_alpha', 'zero', 'one'),
-        blend_equation='func_add',  # see vispy/vispy#2324
-    ),
-    'additive': dict(
-        depth_test=False,
-        cull_face=False,
-        blend=True,
-        blend_func=('src_alpha', 'one'),
-        blend_equation='func_add',
-    ),
-    'minimum': dict(
-        depth_test=False,
-        cull_face=False,
-        blend=True,
-        blend_func=('one', 'one'),
-        blend_equation='min',
-    ),
+    'opaque': {
+        "depth_test": True,
+        "cull_face": False,
+        "blend": False,
+        "blend_func": ('one', 'zero'),
+        "blend_equation": 'func_add',
+    },
```

Comment:
Yeah I'm not a big fan of this. It's a lot clearer to me that this is not data but kwargs if we use `dict()`.

Reply:
I do not understand; could you elaborate?

Reply:
I *think* @brisvag is saying that `dict(kwarg1=value1, kwarg2=value2)` visually looks a lot more like a set of keyword arguments than the dict literal notation. I kind of agree but 🤷 don't feel super strongly.

Reply:
Exactly! Yeah not strongly against this either, I just don't see the benefit (and speedup seems unlikely to be significant...).

Reply:
But do you mean only this file or other places too?

Reply:
I mean almost everywhere where this was changed here (but some make sense, like all the empty `dict()` and others where key aren't "data".) But I'm letting you decide on this, it's really not that annoying :P

Reply:
for me at this moment literal dict is more readable for me, but I earlier switch to this form in other projects.